packages/lesswrong/components/common/TemplateComponent.tsx

```diff
@@ -1,6 +1,6 @@
 // TODO: Import component in components.ts
 import React from 'react';
-import { registerComponent } from '../../lib/vulcan-lib';
+import { registerComponent, Components } from '../../lib/vulcan-lib';
```

Comment:
unnecessary change?

Reply:
I think the TemplateComponent should have Components, you almost always want it

Reply:
sure